### PR TITLE
Devtool doc update

### PR DESCRIPTION
**Pull request by myersCody**
Combination of various tips and tricks shared over the years added to a doc for future developers.

### CODE REVIEW

File: docs/devtools.rst [hunk 1]
```diff
@@ -70,3 +70,105 @@ Example DEVELOPMENT_IDENTITY object::
...
 .. note:: This example is pretty-printed for readability. When setting the enviroment variable, it should be collapsed to one single line.
+
+
+Trino Tips
+----------
+
+Trino is an SQL query engine, but not an SQL database. What does that mean? Traditional databases
+typically consist of a query engine and a storage engine. Trino is just a query engine and does not
+store data. So, inside of koku our download workers will retrieve cost data from the cloud providers
+in the form of CSVs. Then during processing we convert these csv files to parquet
```
Comment (samdoran) [inline docs/devtools.rst:81]:
```suggestion
in the form of CSVs. During processing we convert these csv files to parquet
```

File: docs/devtools.rst [hunk 2]
```diff
@@ -70,3 +70,105 @@ Example DEVELOPMENT_IDENTITY object::
...
+files (Look for the following function in the codebase: `convert_csv_to_parquet`).
```
Comment (samdoran) [inline docs/devtools.rst:82]:
```suggestion
files (Look for the following function in the codebase: ``convert_csv_to_parquet``).
```

File: docs/devtools.rst [hunk 3]
```diff
@@ -70,3 +70,105 @@ Example DEVELOPMENT_IDENTITY object::
...
+
+    S3_ACCESS_KEY=kokuminioaccess
+    S3_SECRET=kokuminiosecret
+
+::
+
+Trino Coordinator
+"""""""""""""""""
+
+This is the brain of the Trino installation, and the node the client connects to submit statements for execution.
```
Comment (samdoran) [inline docs/devtools.rst:122]:
```suggestion
This is the brain of the Trino installation and the node the client connects to when submitting statements for execution.
```

File: docs/devtools.rst [hunk 4]
```diff
@@ -70,3 +70,105 @@ Example DEVELOPMENT_IDENTITY object::
...
+PGPASSWORD=postgres psql postgres -U postgres -h localhost -p 15432
+
+::
+
+Below is a list of Koku commonly used psql commands:
+
+1. Set search Schema: `set search_path=org1234567`
+2. find datatable: `\dt` or `\dt reporting_ocpgcp*`
+3. Tidy up display: `\x auto`
+4. List columns for a table: `\d+ reporting_ocpusagelineitem_daily_summary`
```
Comment (samdoran) [inline docs/devtools.rst:157]:
```suggestion
1. Set search Schema: ``SET search_path TO org1234567``
2. find datatable: ``\dt`` or ``\dt reporting_ocpgcp*``
3. Tidy up display: ``\x auto``
4. List columns for a table: ``\d+ reporting_ocpusagelineitem_daily_summary``
```

File: docs/devtools.rst [hunk 5]
```diff
@@ -70,3 +70,105 @@ Example DEVELOPMENT_IDENTITY object::
...
+Commonly used sql pieces:
+
+Search for Tag Key::
+WHERE pod_labels ? 'environment';
+::
+
+Jinja Templated SQL
+"""""""""""""""""""
+
+We have a development python script that will replace the jinja variables inside of a template file.
```
Comment (samdoran) [inline docs/devtools.rst:168]:
```suggestion
We have a development python script that will replace the Jinja variables inside of a template file.
```

File: docs/devtools.md [hunk 6]
```diff
@@ -75,3 +75,95 @@ Note
 This example is pretty-printed for readability. When setting the
 enviroment variable, it should be collapsed to one single line.
 :::
+
+## Trino Tips
+
+Trino is an SQL query engine, but not an SQL database. What does that mean? Traditional databases
```
Comment (samdoran) [inline docs/devtools.md:81]: Do these paragraphs need to be hard wrapped? It's generally preferable to let the text editor do the wrapping.

File: docs/devtools.md [hunk 7]
```diff
@@ -75,3 +75,95 @@ Note
...
+The coordinator creates a logical model of a query involving a series of stages,
+which is then translated into a series of connected tasks running on a cluster of Trino workers.
+
+You can use the trino UI to see the queries run in trino:
+- http://127.0.0.1:8080/ui/
+
+You can also manually run queries inside of the trino container
+```
+    docker exec -it trino trino --catalog hive --schema org1234567 --user admin --debug
+    SHOW CREATE TABLE <table_name>;
```
Comment (samdoran) [inline docs/devtools.md:133]:
```suggestion
docker exec -it trino trino --catalog hive --schema org1234567 --user admin --debug
SHOW CREATE TABLE <table_name>;
```

**Review by maskarb — APPROVED** [review]
lgtm!

**Pull request merged**